content/java/concepts/errors/terms/negativearraysizeexception/negativearraysizeexception.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'Negativearraysizeexception'
+Description: 'Occurs when an application tries to create an array with negative size.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Errors'
+  - 'Error Handling'
+  - 'Exceptions'
+  - 'Debugging'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The NegativeArraySizeException is a runtime exception in Java that occurs when an application attempts to create an array with a negative size.
+Since the NegativeArraySizeException is an unchecked exception, it does not need to be declared in the throws clause of a method or constructor.
```

Comment:
```suggestion
Since the `NegativeArraySizeException` is an unchecked exception, it does not need to be declared in the throws clause of a method or constructor.
```